cmd/docker/internal/signals/signals_unix.go

```diff
@@ -0,0 +1,9 @@
+package signals
```

Comment:
`_unix` is not recognized in filenames; this should need a manual buildtag: https://github.com/golang/go/issues/51572